cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -21,21 +21,25 @@ metadata:
   # should be in the namespace of the operator
   namespace: rook-ceph
 data:
-  # # (Optional) Ceph Provisioner NodeAffinity.
-  # CSI_PROVISIONER_NODE_AFFINITY: "role=storage-node; storage=rook, ceph"
-  # # (Optional) CEPH CSI provisioner tolerations list. Put here list of taints you want to tolerate in YAML format.
+  # # (Optional) Ceph Provisioner NodeAffinity. e.g., role=storage-node; storage=rook,ceph
+  # # Node affinities are added in <key>=<value>[,<value>]... syntax separated by semicolons (;)
+  # # Affinities are added to CSI pods with `requiredDuringSchedulingIgnoredDuringExecution`
```

Comment:
Can we change CSI pods to CSI provisioner pods. As this applies only to provisioner pod